_data/simple-icons.json

```diff
@@ -5295,6 +5295,21 @@
             "hex": "512DA8",
             "source": "https://favro.com/login"
         },
+        {
+            "title": "FCC",
+            "hex": "1C3664",
+            "source": "https://www.fcc.gov/logos",
+            "guidelines": "https://apps.fcc.gov/oetcf/kdb/forms/FTSSearchResultPage.cfm?id=27980&switch=P",
+            "license": {
+                "type": "custom",
+                "url": "https://www.fcc.gov/licensing"
+            },
```

Comment:
Is there anything actually relevant to the logo on this page?

Reply:
They must have a legal license and meet the requirements from the FCC before using the symbol.